src/lib/components/BlogLink.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+  export let id: string;
+  export let title: string;
+  export let publishedAt: string;
+  export let tags: string[];
+  export let ogpImg: { url: string } | null;
+
+  function formatDate(dateString: string | number | Date) {
+    const date = new Date(dateString);
+    return date.toLocaleDateString("ja-JP", {
+      year: "numeric",
+      month: "2-digit",
+      day: "2-digit",
+    });
+  }
+</script>
+
+<div>
+  <div class="ogp">
+    <a href={`/blogs/${id}`}>
+      {#if ogpImg}
+        <img src={ogpImg.url} alt={title} />
+      {:else}
+        <img src="https://placehold.jp/150x150.png" alt="ogp" />
+      {/if}
+    </a>
+  </div>
+  <p>公開：{formatDate(publishedAt)}</p>
+  <ul>
+    {#each tags as tag (tag)}
+      <li>{tag}</li>
+    {/each}
+  </ul>
+  <a href={`/blogs/${id}`}><h3>{title}</h3></a>
+</div>
+
+<style lang="scss">
+  .ogp {
+    width: 100%;
+    aspect-ratio: 100 / 52.5;
+    border-radius: 5px;
```

Comment:
定義が分かりにくて申し訳ないですが, SCSS 変数を使ってあげるなら
```suggestion
    border-radius: $radii-md;
```
が近いかも？